admin/sql/create_tables.sql

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
in a few places you have added some extra whitespace. You should be careful that you, or your editor doesn't do this. See if there is a setting that might be causing this